CRM/Contact/Form/Edit/Email.php

```diff
@@ -68,16 +68,12 @@ public static function buildQuickForm(&$form, $blockCount = NULL, $blockEdit = F
       $multipleBulk = CRM_Core_BAO_Email::isMultipleBulkMail();
 
       //On-hold select
-      if ($multipleBulk) {
-        $holdOptions = array(
-          0 => ts('- select -'),
-          1 => ts('On Hold Bounce'),
-          2 => ts('On Hold Opt Out'),
-        );
-        $form->addElement('select', "email[$blockId][on_hold]", '', $holdOptions);
+      $holdOptions = CRM_Core_BAO_Email::getHoldOptions();
```

Comment:
I am feeling even sadder - we have a boolean field & we just stick option values in it anyway :-(